JavaScript/recognizers-date-time/src/dateTime/chinese/holidayConfiguration.ts

```diff
@@ -0,0 +1,259 @@
+import { IHolidayExtractorConfiguration, BaseHolidayParserConfiguration, BaseHolidayParser } from "../baseHoliday"
+import { RegExpUtility, ExtractResult, Match, IExtractor, IParser, StringUtility, ChineseIntegerExtractor, AgnosticNumberParserFactory, AgnosticNumberParserType, ChineseNumberParserConfiguration } from "recognizers-text-number";
+import { Constants as NumberConstants } from "recognizers-text-number"
+import { DateUtils, FormatUtil, DateTimeResolutionResult } from "../utilities";
+import { ChineseDateTime } from "../../resources/chineseDateTime";
+import { IDateTimeParser, DateTimeParseResult } from "../parsers";
+import { Constants, TimeTypeConstants } from "../constants";
+
+export class ChineseHolidayExtractorConfiguration implements IHolidayExtractorConfiguration {
+    readonly holidayRegexes: RegExp[]
+            
+    constructor() {
+        this.holidayRegexes = [
+            RegExpUtility.getSafeRegExp(ChineseDateTime.HolidayRegexList1),
+            RegExpUtility.getSafeRegExp(ChineseDateTime.HolidayRegexList2),
+            RegExpUtility.getSafeRegExp(ChineseDateTime.LunarHolidayRegex)
+        ];
+    }
+}
+
+class ChineseHolidayParserConfiguration extends BaseHolidayParserConfiguration {
+    constructor() {
+        super();
+        this.holidayRegexList = [
+            RegExpUtility.getSafeRegExp(ChineseDateTime.HolidayRegexList1),
+            RegExpUtility.getSafeRegExp(ChineseDateTime.HolidayRegexList1)
+        ];
+        this.holidayFuncDictionary = this.initHolidayFuncs();
+        this.variableHolidaysTimexDictionary = ChineseDateTime.HolidayNoFixedTimex;
+    }
+
+    getSwiftYear(source: string): number {
+        if (source.endsWith('年')) return 0;
+        if (source.endsWith('去年')) return -1;
```

Comment:
Ideally these string values should be moved to the resource files. We can have someone with Chinese knowledge do it.